docker/etc/nginx/custom/lua/file_too_large_ajax.lua

```diff
@@ -0,0 +1,12 @@
+local cjson = require "cjson"
+
+local message = '<div class="" data-drupal-messages=""><div class="messages__wrapper"> <div class="messages messages--error cd-alert cd-alert--error" role="alert" data-drupal-message-type="error"> <div role="alert" aria-label="error"> <svg class="cd-icon cd-icon--error" aria-hidden="true" focusable="false" width="24" height="24"> <use xlink:href="#cd-icon--error"></use> </svg> <div class="cd-alert__container cd-max-width"> <div class="cd-alert__message [ cd-flow ]"> File Too Large. The file exceeds the maximum size of 25 MB. </div> <div class="cd-alert__title"> <h2 class="visually-hidden">Error message</h2> </div>  </div> </div> </div>'
```

Comment:
Hmm, this hardcodes the max file size which depends on the version of the form (submitter = 25MB currently, probably will be reduced to fit the AWS Nova model limitations, 100MB for editors etc.).

Also the 413 can happened for different fields non only the file attachment one (ex: report image field).

Might be better to add a proper Drupal route with a controller and return the full page or the error code depending on whether its from ajax or not by checking Request::isXmlHttpRequest(). We could check if the current user is a submitter for example and retrieve the current file limit etc. and adjust based on the other info we can get from the request.

Reply:
Sure, but that's not MVP :-)

Reply:
But nobody is using the document drop currently so no need to rush on that...